pkg/server/commands/reverseexpand/reverse_expand.go

```diff
@@ -325,7 +345,10 @@ LoopOnEdges:
 					Relation: innerLoopEdge.TargetReference.GetRelation(),
 				},
 			}
-			atomic.AddUint32(resolutionMetadata.DispatchCount, 1)
+			_ = atomic.AddUint32(resolutionMetadata.DispatchCount, 1)
```

Comment:
curious why do we need `_ =`

Reply:
Also, if we are going to use atomic.AddUint32, why not set the variable as atomic.Uint32 instead?  

In addition, instead of doing the adding here and then loading the variable in the throttle function, why not do the adding in the throttle function (though you will need to move the check enable to inside the throttle function).

Reply:
The reason is since resolution Metadata will be required for remote dispatching and to keep it network compliant, I avoided using atomic values over there